### DISCUSSION TITLE
How can I see upstream request headers?

### DISCUSSION THREAD

**Discussion by tirelibirefe** [Kong/kong#12057] [Help] [answered]
Hello,
Nowadays I have troubles with "request transformer".
Although I enabled debug logging, I couldn't see upstream request headers. How can I see upstream request headers?

Thanks & Regards

**Comment by StarlightIbuki** [comment, marked as answer]
Please refer to https://docs.konghq.com/gateway/latest/plugin-development/pdk/kong.service.request/
